src/main/java/selenium/pages/FirstExercisePage.java

```diff
@@ -0,0 +1,42 @@
+package selenium.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FirstExercisePage extends Exercise{
+
+    public FirstExercisePage(WebDriver driver) {
+        super(driver);
+    }
+
+    private List<String> getButtonNumbers(){
+        List<WebElement> instructions = driver.findElements(By.xpath("//td/code"));
```

Comment:
zrobilbym to explicit waitem - zobacz, jak strona bedzie wolniej dzialac, to dostaniesz bledy bo nie wczyta tej listy

Reply:
done